source/reference/verification.txt

```diff
@@ -99,11 +99,14 @@ Migration Verifier connects to the source and destination clusters and performs
 a series of verification checks, comparing documents, views, and indexes to
 confirm the sync was successful.
 
+Migration Verifier does not support DDL operations. When using the Migration
+Verifier, ensure that your source cluster does not use DDL commands.
```

Comment:
[question]

Is there a way to get more specific about what using DDL commands means in this context? 

Are we saying users shouldn't run DDL operations on the source cluster during migration? Are there additional considerations?

Reply:
Thanks for taking a look at this and for raising this questions, @jmd-mongo ! I tried to reword this sentence now to clarify that no DDL commands can be running on the source cluster when the user is verifying their data using Migration Verifier – lmk if you have any changes or suggestions!

I don't believe there are any additional considerations – the Limitations section in the [Migration Verifier GitHub ReadMe](DOCSP-43721-migration-verifier-disclaimer) I _think_ gives the user all the specifics they'd need for this consideration.